lib/fibers_stack/SFibersStackHitFinderPar.h

```diff
@@ -38,9 +41,15 @@ class SIFI_EXPORT SFibersStackHitFinderPar : public SPar
     Float_t getLambda() const { return fLambda; }
     Float_t getAlpha() const { return fAlpha; }
 
+    Float_t getResPos(Int_t i) const;
+    Float_t getResEne(Int_t i) const;
+
     void setA0(Float_t v) { fA0 = v; }
     void setLambda(Float_t v) { fLambda = v; }
     void setAlpha(Float_t v) { fAlpha = v; }
+
+    void setResPos(TArrayF f) { fResPos = f; }
```

Comment:
These two require proper set functions similar to set. This way you do not assure that size of f is equal to 3.